### PR TITLE
Added page header style configuration

### PR DESCRIPTION
**Pull request by wadeking98**
Signed-off-by: wadeking98 <wkingnumber2@gmail.com>

# Summary of Changes

Added the ability to customize the style of the page header in the themes file

# Related Issues

N/A

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: core/App/navigators/defaultStackOptions.ts [hunk 1]
```diff
@@ -1,18 +1,22 @@
 import { Theme } from '../theme'
 
-export function createDefaultStackOptions({ ColorPallet }: Theme) {
+export function createDefaultStackOptions(theme: Theme) {
```
Comment (jleach) [inline core/App/navigators/defaultStackOptions.ts]: @wadeking98 Why was the change needed? I think the previous method of deconstructing was okish. Just asking - not a value statement.

Comment (wadeking98) [reply core/App/navigators/defaultStackOptions.ts]: I needed to access properties outside of the ColourPallete like the textTheme in order to apply the BCSans-Regular font to the headings

Comment (amanji) [reply core/App/navigators/defaultStackOptions.ts]: Perhaps just add `TextTheme` to the destructuring assignment. Will also make it clear you only need those fields.

Comment (wadeking98) [reply core/App/navigators/defaultStackOptions.ts]: change completed

**Review by amanji — APPROVED** [review]

**Pull request merged**